src/main/java/cafeboard/board/BoardService.java

```diff
@@ -0,0 +1,66 @@
+package cafeboard.board;
+
+import cafeboard.board.DTO.BoardResponse;
+import cafeboard.board.DTO.BoardUpdate;
+import cafeboard.board.DTO.CreateBoard;
+import cafeboard.board.DTO.BoardDetailedResponse;
+import cafeboard.post.DTO.PostResponse;
+import cafeboard.post.Post;
+import cafeboard.post.PostRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@Service
+public class BoardService {
+
+    private final BoardRepository boardRepository;
+    private final PostRepository postRepository;
+
+    public BoardService(BoardRepository boardRepository, PostRepository postRepository) {
+        this.boardRepository = boardRepository;
+        this.postRepository = postRepository;
+    }
+
+    public BoardDetailedResponse create(CreateBoard dto) {  // 게시판 생성
+        Board board = new Board(dto.title());
+        boardRepository.save(board);
+        return new BoardDetailedResponse(board.getId(), board.getTitle(), board.getCreatedTime());
+    }
+
+    public List<BoardResponse> findAll() { // 모든 게시판 조회
+        List<Board> boards = boardRepository.findAll();
+        return boards.stream().map(board -> new BoardResponse(board.getId(), board.getTitle())).toList();
```

Comment:
사소한 거긴 한데 아래처럼 줄을 나누어 주는 것이 가독성 관점에서 더 좋다고 인식돼요. 한 줄에 점(`.`) 하나씩이라고 보시면 돼요.
```suggestion
        return boards.stream()
                .map(board -> new BoardResponse(board.getId(), board.getTitle()))
                .toList();
```